src/django/pfb_network_connectivity/filters.py

```diff
@@ -13,6 +13,16 @@ class OrgAutoFilterBackend(filters.BaseFilterBackend):
     """Filter that only allows users to see their organization's own objects."""
 
     def filter_queryset(self, request, queryset, view):
+        if queryset.model == Organization:
+            return queryset.filter(uuid=request.user.organization_id)
+        else:
+            return queryset.filter(organization=request.user.organization)
+
+
+class OrgOrAdminAutoFilterBackend(filters.BaseFilterBackend):
+    """Filter that allows non-admin users to see only their own organization's objects."""
+
+    def filter_queryset(self, request, queryset, view):
```

Comment:
This change surfaced an existing bug in the AnalysisJob list--AnalysisJob doesn't have an `organization` attribute.  Nor should it, since it's foreign keyed to Neighborhood, which does, so it would be redundant.  But that means filtering on `organization` in these two filters throws an error (which was hidden before for users from the admin org, since they got the unfiltered results).  Simplest thing is probably to add another special clause, i.e.
```
if queryset.model == AnalysisJob:
    return queryset.filter(neighborhood__organization=request.user.organization)
```
Though it would be nice if there were a more general way to do it.